pkg/controller/statefulset/stateful_pod_control.go

```diff
@@ -301,15 +308,22 @@ func (spc *StatefulPodControl) recordPodEvent(verb string, set *apps.StatefulSet
 // nil the generated event will have a reason of v1.EventTypeNormal. If err is not nil the generated event will have a
 // reason of v1.EventTypeWarning.
 func (spc *StatefulPodControl) recordClaimEvent(verb string, set *apps.StatefulSet, pod *v1.Pod, claim *v1.PersistentVolumeClaim, err error) {
+	spc.recordUnavailableClaimEvent(verb, set, pod, claim.Name, err)
```

Comment:
Emm.. it's just werid to have an `err` var as a method argument, why we need wrap this func body as `recordUnavailableClaimEvent`